raw/mock_exam/docs/questions/question-32.adoc

```diff
@@ -0,0 +1,105 @@
+// tag::DE[]
+=== Frage 32
+**ID: Q-20-04-25**
```

Comment:
Antwortmöglichkeiten weichen stark von der ursprünglichen Frage in den Mock Exams ab. Evtl. kenntlich machen.

Reply:
don't really understand that comment.  Imho the answer options fit the question